app/settings.html

```diff
@@ -23,13 +19,9 @@
       </p>
       <h2>breaks</h2>
       <p>
-        <span id="breakDurationMinus" class="pointer">-</span>
-        <span id="breakDuration"></span>
-        <span id="breakDurationPlus" class="pointer">+</span>
+        <input type="number" id="breakDuration" min="0" max="59" step="1" value=5>
```

Comment:
IMO, the text should be different with something clickable, or else it's confusing.